doc/main/tbb_userguide/Guiding_Task_Scheduler_Execution.rst

```diff
@@ -61,7 +61,8 @@ assign a NUMA node identifier to the ``task_arena::constraints::numa_id`` field.
 
 The processors with `Intel® Hybrid Technology <https://www.intel.com/content/www/us/en/products/docs/processors/core/core-processors-with-hybrid-technology-brief.html>`_
 contain several core types, each is suited for different purposes.
-For example, some applications may improve their performance by preferring execution on the most performant cores.
+In most cases, hybrid systems don't require special API usage but in special cases, the performance
+may be improved by setting the preferred core type.
```

Comment:
Just a suggestion:

In most cases, systems with hybrid CPUs shows reasonable performance without involving additional APIs calls, however in some exceptional cases, performance my by tuned by setting the preferred core type.

Reply:
```suggestion
In most cases, systems with hybrid CPU architecture show reasonable performance without involving additional API calls,
however in some exceptional situations, performance may be tuned by setting the preferred core type.
```
@ivankochin  Thank you for the suggestion :)
I guess it looks better although I made a few edits (still don't like the repeated "cases" here). What do you think?

Reply:
Yeah, that's better, but I would like to propose "scenarios" instead of "situations"